wamv_gazebo/config/rviz_vrx.rviz

```diff
@@ -47,9 +43,9 @@ Visualization Manager:
       Name: Grid
       Normal Cell Count: 0
       Offset:
-        X: 0
-        Y: 0
-        Z: 0
+        X: 532
+        Y: -162
+        Z: -0.11999999731779099
```

Comment:
Instead of changing the offset, can we set the reference frame to wamv/baselink? Then it should center the grid around the vehicle instead of at the origin of the world.

Reply:
@M1chaelM fixed